src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -50,11 +51,12 @@ public sealed partial class CSharpCompilation : Compilation
         // !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
         private readonly CSharpCompilationOptions _options;
-        private readonly Lazy<UsingsFromOptionsAndDiagnostics> _usingsFromOptions;
-        private readonly Lazy<ImmutableArray<NamespaceOrTypeAndUsingDirective>> _globalImports;
-        private readonly Lazy<Imports> _previousSubmissionImports;
-        private readonly Lazy<AliasSymbol> _globalNamespaceAlias;  // alias symbol used to resolve "global::".
-        private readonly Lazy<ImplicitNamedTypeSymbol?> _scriptClass;
+        private UsingsFromOptionsAndDiagnostics? _usingsFromOptions;
```

Comment:
>_usingsFromOptions [](http://example.com/codeflow?start=49&length=18)

We usually add "lazy" prefix to lazy initialized fields. For example for this one it will look like ""_lazyUsingsFromOptions```. This comment is applicable to other fields affected in this PR.